src/graphql/main/queries.ts

```diff
@@ -27,6 +28,7 @@ const fields = {
   btcPrice: BtcPriceQuery,
   btcPriceList: BtcPriceListQuery,
   onChainTxFee: OnChainTxFeeQuery,
+  onChainUsdTxFee: OnChainUsdTxFeeQuery,
```

Comment:
you are missing shield update, this query should use isAuthenticated in graphql-main-server

Reply:
Good catch! Done: https://github.com/GaloyMoney/galoy/pull/2072/commits/400ba13a8c53504207c1c21484874e6538c07e00